vnext/ReactUWP/Modules/DevSupportManagerUwp.cpp

```diff
@@ -73,12 +74,67 @@ void DevSupportManager::LaunchDevTools(const facebook::react::DevSettings &setti
   DownloadFromAsync(facebook::react::DevServerHelper::get_LaunchDevToolsCommandUrl(settings.debugHost)).get();
 }
 
+std::future<void> DevSupportManager::CreatePackagerConnection(const facebook::react::DevSettings &settings) {
+  m_ws = winrt::Windows::Networking::Sockets::MessageWebSocket();
+
+  m_wsMessageRevoker = m_ws.MessageReceived(
+      winrt::auto_revoke, [reloadCallback = settings.liveReloadCallback](auto && /*sender*/, auto &&args) noexcept {
+        try {
+          auto reader = args.GetDataReader();
+
+          uint32_t len = reader.UnconsumedBufferLength();
+          if (args.MessageType() == winrt::Windows::Networking::Sockets::SocketMessageType::Utf8) {
+            reader.UnicodeEncoding(winrt::Windows::Storage::Streams::UnicodeEncoding::Utf8);
+            std::vector<uint8_t> data(len);
+            reader.ReadBytes(data);
+
+            auto response =
+                std::string(Microsoft::Common::Utilities::CheckedReinterpretCast<char *>(data.data()), data.size());
+            auto json =
+                winrt::Windows::Data::Json::JsonObject::Parse(Microsoft::Common::Unicode::Utf8ToUtf16(response));
```

Comment:
could use folly::parseJSON here instead but this is fine too

Reply:
Trying to avoid adding extra folly usage, since long term once we are off the legacy bridge, we might be able to ditch folly altogether.